### DISCUSSION TITLE
Unable to fetch Delay Changes

### DISCUSSION THREAD

**Discussion by mahrukhS** [InternetHealthReport/gsoc#29] [Q&A] [answered]
There is a Python script to fetch delay changes on the [documentation](https://ihr.iijlab.net/ihr/en-us/documentation) page. 
Would anyone like to confirm if this script works since it is not fetching at my end? 
I tried using a different URL and recent dates
Added below is the script and error image

## Script
![image](https://github.com/InternetHealthReport/gsoc/assets/66876079/2fc7456c-d2f4-4db2-9449-dd06d046012a)

## Error
![image](https://github.com/InternetHealthReport/gsoc/assets/66876079/5b0e295e-2caa-4daa-80de-1e734bcc0515)

**Comment by romain-fontugne** [comment, marked as answer]
Ah, thanks for reporting that the documentation is outdated. There is now a limit of 7 days, you cannot query more than 7 days at a time.
Can you try again with for example 2023/11/01 to 2023/11/03?

**Reply by mahrukhS** [reply]
Thanks. Yes, there is a limitation of 7 days. 
I am still trying to figure out how to access data using REST API. 
The Python Library method to access data works perfectly, though.

![image](https://github.com/InternetHealthReport/gsoc/assets/66876079/9038c4de-4934-4d7f-ab12-ffd535998631)
